src/yashCJun20/Count.java

```diff
@@ -0,0 +1,47 @@
+package yashCJun20;
+
+public class Count {
+	
+	void count (String s){
+		
+		int digitCnt=0;
+		int letterCnt=0;
+		int upperCaseCnt=0;
+		int lowerCaseCnt=0;
+		int specialCharCnt=0;
+		
+		for (int i=0;i<s.length();i++) {
+			char ch = s.charAt(i);
+			
+			if(Character.isDigit(ch)==true)
+				digitCnt++;
+			else if(Character.isLetter(ch)==true)
+				letterCnt++;
```

Comment:
letterCnt without using this variable and else if block you can achieve expected output.